netmaster/master/network.go

```diff
@@ -421,6 +473,13 @@ func DeleteNetworkID(stateDriver core.StateDriver, netID string) error {
 		return err
 	}
 
+	err = detachServiceContainer(netID, nwCfg.Tenant)
```

Comment:
Should this code block be before line 468 above? Otherwise if reading tenant fails for some reason then we won't be able to do this cleanup as well.